bids/layout/tests/conftest.py

```diff
@@ -40,28 +40,26 @@ def layout_ds005_derivs():
     return layout
 
 
-@pytest.fixture(scope="module")
-def layout_ds005_multi_derivs():
+fdir = tempfile.mkdtemp()
```

Comment:
Is this intended to be common to both tests? Can we not use the `tmp_path` fixture?

Reply:
Yeah it is. Didn't know about that fixture.